.eslintrc.json

```diff
@@ -47,6 +47,7 @@
           "error",
           {
             "enforceBuildableLibDependency": true,
+            "checkDynamicDependenciesExceptions": [".*"],
```

Comment:
This is practically disabling the rule. I would rather:
- fix what should be fixed (e.g. spec files) and
- use comment toggles to switch off others